aptos-move/aptos-replay-benchmark/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "aptos-replay-benchmark"
```

Comment:
Yes, I think our tooling would benefit from more unified CLI features (e.g., here, comparison-e2e-testing can re-use `Diff`s, etc.). For gas it is a bit tricky here because gas profiler is created inside the VM, so block executor does not have any context (unless we create a special entry point).

I would say in the future we would want to unify this, log gas/sec in addition to pure time, etc.